### PR TITLE
BEAM-2578 DebuggingWordCountTest Fails on Windows

### PR DESCRIPTION
**Pull request by eralmas7**
[INFO] --- maven-failsafe-plugin:2.20:integration-test (default) @ beam-examples-java ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-dependency-plugin:3.0.1:analyze-only (default) @ beam-examples-java ---
[INFO] No dependency problems found
[INFO] 
[INFO] --- maven-failsafe-plugin:2.20:verify (default) @ beam-examples-java ---
[INFO] Tests are skipped.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:41 min
[INFO] Finished at: 2017-07-10T23:18:01+05:30
[INFO] Final Memory: 45M/129M
[INFO] ------------------------------------------------------------------------

### CODE REVIEW

File: examples/java/src/test/java/org/apache/beam/examples/DebuggingWordCountTest.java [hunk 1]
```diff
@@ -35,6 +35,14 @@
 public class DebuggingWordCountTest {
   @Rule public TemporaryFolder tmpFolder = new TemporaryFolder();
 
+  private String getFilePath(String filePath) {
+      if (filePath.contains(":")) {
+          String restPath = filePath.replace("\\", "/").split(":")[1];
+          return restPath;
```
Comment (flaviocf) [inline examples/java/src/test/java/org/apache/beam/examples/DebuggingWordCountTest.java]: You can also merge these lines into:
return filePath.replace(...

Comment (eralmas7) [inline examples/java/src/test/java/org/apache/beam/examples/DebuggingWordCountTest.java]: Thanks flaviocf. Have fixed it.

**Comment by jbonofre** [comment]
R: @jbonofre

File: examples/java/src/test/java/org/apache/beam/examples/DebuggingWordCountTest.java [hunk 2]
```diff
@@ -35,6 +35,13 @@
 public class DebuggingWordCountTest {
   @Rule public TemporaryFolder tmpFolder = new TemporaryFolder();
 
+  private String getFilePath(String filePath) {
+      if (filePath.contains(":")) {
+          return filePath.replace("\\", "/").split(":")[1];
```
Comment (jbonofre) [inline examples/java/src/test/java/org/apache/beam/examples/DebuggingWordCountTest.java]: It sounds good to me. You can also use NIO `Paths.get()` that automatically does the conversion.

**Pull request closed without merging**